src/views/SimpleAccount/DepositLists.tsx

```diff
@@ -38,24 +38,26 @@ export const SimpleV4DepositList: React.FC<{ usersAddress: string }> = (props) =
 }
 
 /**
- *
+ * Displays V3 deposits (hidden after loading if empty).
  * @param props
  * @returns
  */
 export const SimpleV3DepositList: React.FC<{ usersAddress: string }> = (props) => {
   const { usersAddress } = props
-  const { data } = useUsersV3PrizePoolBalances(usersAddress, false)
```

Comment:
Why remove the filtering by passing `false`?

Reply:
`V3DepositsList` used to re-query this data without `false` (searching for more pools I imagine) - now that we're only querying once I assumed we should include those pools.

If those aren't required at all, then adding `false` could speed up that query for sure.

Reply:
Nope! It's all the same massive query then just filters from the query results. Without the `false` we'll filter peoples ETH/POOL & PPOOL balances from the balances list. It's a _little_ awkward since we'll render PPOOL twice but one is a link out, it's fine for now.